.changeset/quick-seas-pay.md

```diff
@@ -0,0 +1,6 @@
+---
+"@cmpsr/markdown-editor": minor
+"@cmpsr/components": minor
```

Comment:
nit: we should have a different changeset for the components package

Reply:
The components package has been bumped because it will now export a new icon. 
A patch or major bump doesn't seem to be the best option. Besides, [setting it to path caused the canary release to fail](https://github.com/cmpsr/composer/actions/runs/6459936082/job/17536698277?pr=818). So I've rolled it back to `minor`.

Reply:
sorry I do not explain properly myself, what I meant is that we can have two different changeset individually generated for each package so the changelog of the package component reflects the proper change message instead of the message of the markdown editor